internal/common/basic.go

```diff
@@ -30,11 +30,11 @@ func runBasic() {
 			}
 		})
 
-		It("should be able to connect to MC cluster", func() {
+		It("should be able to connect to MC cluster", FlakeAttempts(3), func() {
```

Comment:
Oh nice, I didn't even know this exists in Ginkgo!

Reply:
Yeah it's pretty nice. Rather than having a full retry of the whole suite 😄 

I was reluctant to use it initially as ideally don't want flakes to even exist in the tests but networking is what it is 😅